monitor.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ebiten
+
+import (
+	"github.com/hajimehoshi/ebiten/v2/internal/ui"
+)
+
+// Monitor represents a monitor available to the system.
+type Monitor = ui.Monitor
```

Comment:
As an alias is used, there is no API documentation about `Monitor`. We should define `Monitor`'s functions in this package, though this might be redundant.